dbt/include/databricks/macros/materializations/seed.sql

```diff
@@ -29,7 +29,7 @@
           {%- endfor %}
       {% endset %}
 
-      {% do adapter.add_query(sql, bindings=bindings, abridge_sql_log=True) %}
+      {% do adapter.add_query(sql, bindings=bindings, abridge_sql_log=True, close_cursor=True) %}
```

Comment:
Why do we need to pass the `close_cursor` flag for seed?

Reply:
`adapter.add_query` doesn't close the cursor by default, then the cursor will remain opened.

Reply:
I see!